### ISSUE TITLE
Localhost-limits

### ISSUE THREAD

**Issue by ineiti** [dedis/cothority#17]
I tried to run the localhost-simulation with 256 collective signers and it stopped without any comment...

**Comment by ineiti** [comment]
Find out where the error happens and at least display that something went wrong.

**Comment by ineiti** [comment]
Probably a port-collision - should this be checked upon?

**Comment by Liamsi** [comment]
This is on the `development` branch, right?

**Comment by Liamsi** [comment]
To reproduce this do:
1) edit `deploy/simulation/sign_single_toml`
```
@@ -1,5 +1,5 @@
 Machines = 8
 App = "sign"
 
 ppm, bf, rounds, rate, suite
-256, 8, 20, 10, "Ed25519"
+32, 8, 20, 10, "Ed25519"
```
2) And than run:
```
$ cd deploy
$ go build 
$ ./deploy simulation/sign_single.toml 
```

**Comment by Liamsi** [comment]
Should be fixed in #159.

**Comment by ineiti** [comment]
Done
